weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/tabularListViews/operators.ts

```diff
@@ -3,6 +3,8 @@ import _ from 'lodash';
 
 import {Query} from '../../wfReactInterface/traceServerClientInterface/query';
 
+const FIELDS_NO_FLOAT_CONVERT = ['summary.weave.latency_ms'];
```

Comment:
Let's add the explanation you put in the PR description here as a comment.

Reply:
yeah good idea